src/lib/chips/_chips-theme.scss

```diff
@@ -13,6 +13,11 @@
   }
 
   .md-chip.selected {
+    background-color: #808080;
```

Comment:
I can definitely do that; but do we have any specs for a dark theme? I think it would be nice to change it as the current chips in dark mode do look a bit out of place.